building-and-research/human-in-the-loop.md

```diff
@@ -0,0 +1,45 @@
+---
+nav_order: tbd
+parent: Building and research
+title: Human-in-the-loop
+description: Machine translation supported by human post-editing or evaluation
+---
+
+## Definition
+
+Human-in-the-loop consists of using human feedback for additional training of translation engines. Feedback is provided in one of the following ways:  
```

Comment:
As per our style guide, let's bold the subject matter.

---
**Human-in-the-loop** consists of...